aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -387,6 +383,90 @@ impl AptosVM {
         }
     }
 
+    fn finish_aborted_transaction(
+        &self,
+        gas_meter: &mut impl AptosGasMeter,
+        txn_data: &TransactionMetadata,
+        resolver: &impl AptosMoveResolver,
+        log_context: &AdapterLogSchema,
+        change_set_configs: &ChangeSetConfigs,
+    ) -> Result<(VMChangeSet, FeeStatement), VMStatus> {
+        // storage refund is zero since no slots are deleted in aborted transactions.
+        let storage_refund = 0;
```

Comment:
nit: `const ZERO_STORAGE_REFUND`?

Reply:
@ziaptos do you think let is better than local const? why?

Reply:
local constants? if that exists... we should try... otherwise I'm more aligned with just nerfing this and leaving a comment.